[ErrorException] ZipArchive::statIndex(): Invalid or uninitialized Zi…

Forestsoft-de opened the pull request:
…p object

make a useful error message to determine license related configuration problems.

Forestsoft-de committed:
[ErrorException] ZipArchive::statIndex(): Invalid or uninitialized Zip object

make a useful error message to determine license related configuration problems.

shyim commented on src/Shyim/PluginInstaller.php:
```diff
@@ -207,14 +229,21 @@ private static function apiRequest($path, $method, array $params = [])
-        curl_close($ch);
+        $content = self::makePluginHTTPRequest($url);
+
+        if (substr($content, 0, 1) == "{" && substr($content, strlen($content) - 1, 1) == "}") {
```
Could we not check here the response Content-Type?

Forestsoft-de replied:
sure of course. ive just updated the pr.

shyim commented:
I have created an internal Ticket to add in that Test Account an expired test version. So we can add also a Test-Case.

Forestsoft-de committed:
Merge branch 'master' of https://github.com/shyim/store-plugin-installer

Forestsoft-de committed:
[ErrorException] ZipArchive::statIndex(): Invalid or uninitialized Zip object

make a useful error message to determine license related configuration problems.

Implement check of content type and return json array.
Make the tests great again by implementation of config mock for new feature.
